### PR TITLE
Added Longest Alternating Subarray

### LINKED ISSUE #1174: Length of Longest Alternating Subarray of an array in C, C++, Java, Python, Go.

### PR DESCRIPTION
**Pull request by singh-shreya6**
Fixes #1174 

#### Checklist:
- [x] 4 space indentation.
- [x] Coding conventions are followed.
- [x] Input is taken dynamically.
- [x] Sample Input / Output is added at the end of file.
- [x] Logic Documentation (Comments).
- [x] File names are correct.

#### Changes proposed in this pull request:
- Added Longest Alternating Subarray in 5 languages

#### Languages Used:
- C
- C++
- Python
- Java
- Go

#### Files Added:
- Longest_Alternating_Subarray.py
- Longest_Alternating_Subarray.c
- Longest_Alternating_Subarray.go
- Longest_Alternating_Subarray.java
- Longest_Alternating_Subarray.cpp

### CODE REVIEW

File: Longest_Alternating_Subarray/Longest_Alternating_Subarray.py [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+        count = 1
+print(result)
+
+'''
+Input :
+num = 4
+Array = [-1, -4, -5, 20]
+
+Output :
+
```
Comment (somya-kapoor) [inline Longest_Alternating_Subarray/Longest_Alternating_Subarray.py]: Please remove the line 33

File: Longest_Alternating_Subarray/Longest_Alternating_Subarray.java [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+    }
+}
+
+/*
+Input :
+num = 4
+Array = [-1, -4, -5, 20]
+
+Output :
+
```
Comment (somya-kapoor) [inline Longest_Alternating_Subarray/Longest_Alternating_Subarray.java]: Please remove the line 47

File: Longest_Alternating_Subarray/Longest_Alternating_Subarray.go [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+    fmt.Print(result)
+}
+
+/*
+Input :
+num = 4
+Array = [-1, -4, -5, 20]
+
+Output :
+
```
Comment (somya-kapoor) [inline Longest_Alternating_Subarray/Longest_Alternating_Subarray.go]: Please remove the line 48

File: Longest_Alternating_Subarray/Longest_Alternating_Subarray.cpp [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+    cout << result;
+}
+
+/*
+Input :
+num = 4
+Array = [-1, -4, -5, 20]
+
+Output :
+
```
Comment (somya-kapoor) [inline Longest_Alternating_Subarray/Longest_Alternating_Subarray.cpp]: Please remove the line 44

**Comment by singh-shreya6** [comment]
@somya-kapoor Those blank lines are just provided for clarity. It is just one blank line in comments. Should it make any difference ? 
Please review.

**Comment by somya-kapoor** [comment]
@singh-shreya6  I am talking about the lines between output and output statement ,there is no need for that in my opinion.

**Comment by singh-shreya6** [comment]
Hi @somya-kapoor , 
I am also talking about that. It is just for clarity. In comments, it is more important to give clarity and it doesn't alter with conventions. So I think it is okay. Please review.

**Pull request merged**